services/gundeck/src/Gundeck/Presence/Data.hs

```diff
@@ -7,22 +7,19 @@ module Gundeck.Presence.Data
     , deleteAll
     ) where
 
-import Control.Monad
+import Imports
 import Control.Monad.Catch
 import Data.Aeson
 import Data.ByteString.Builder (byteString)
 import Data.ByteString.Conversion hiding (fromList)
-import Data.ByteString (ByteString, isPrefixOf)
-import Data.Foldable (for_)
 import Data.Id
-import Data.Maybe (mapMaybe)
 import Data.Misc (Milliseconds)
-import Data.Monoid
 import Database.Redis.IO hiding (Milliseconds)
 import Gundeck.Monad (Gundeck, posixTime)
 import Gundeck.Types
 import Gundeck.Util.Redis
 
+import qualified Data.ByteString as Strict
```

Comment:
`SBS` instead of `Strict`?  not important.

another idea:

```haskell
$(importQualifiedDefaults)
```

Which expands to a list of qualified imports we usually want.  I guess the problem is that this would trigger `unused import` warnings?  Also not important.  :-)  And we can always do that another time.

Reply:
TH can't generate imports

Reply:
this problem (preludes can't provide qualified imports) has no solution AFAIK; you can use `#include`s but then you have to turn off the "unused imports" warning

Reply:
Yes, i've used CPP in my last project, and the results are inconclusive.  I think the other(s) liked it, I was skeptical.

Ok, no default qualified imports for now then!